parm/config/config.resources

```diff
@@ -467,6 +467,81 @@ elif [ $step = "eupd" ]; then
         export memory_eupd="3072M"
     fi
 
+elif [ $step = "eupdfsoi" ]; then
+
+    export wtime_eupdfsoi="03:00:00"
+    if [ $CASE = "C768" ]; then
+      export npe_eupdfsoi=960
+      export nth_eupdfsoi=6
+      if [[ "$machine" = "WCOSS_DELL_P3" ]]; then
+        export nth_eupdfsoi=7
+      fi
+      if [[ "$machine" = "HERA" ]]; then
+        export npe_eupdfsoi=150
+        export nth_eupdfsoi=40
+      fi
+    elif [ $CASE = "C384" ]; then
+      #export npe_eupdfsoi=270
+      #export nth_eupdfsoi=2
+      export npe_eupdfsoi=400
+      export nth_eupdfsoi=5
+      if [[ "$machine" = "WCOSS_DELL_P3" ]]; then
+        export nth_eupdfsoi=9
+      fi
+      if [[ "$machine" = "HERA" ]]; then
+        export npe_eupdfsoi=100
+        export nth_eupdfsoi=40
+      fi
+    elif [ $CASE = "C192" -o $CASE = "C96" -o $CASE = "C48" ]; then
+      export npe_eupdfsoi=42
+      export nth_eupdfsoi=2
+      if [[ "$machine" = "HERA" ]]; then
+        export npe_eupdfsoi=40
+        export nth_eupdfsoi=40
+      fi
+    fi
+    export npe_node_eupdfsoi=$(echo "$npe_node_max / $nth_eupdfsoi" | bc)
+    if [[ "$machine" == "WCOSS_C" ]]; then
```

Comment:
We've dropped support for the Crays (WCOSS_C) so please remove the new WCOSS_C lines that have been added. Thanks!